create_pdfs.py

```diff
@@ -97,32 +113,40 @@ def main():
             name_list = NAMES
 
         for name in name_list:
-            cert = CertificateGen(
-                course,
-                args.template_file,
-                aws_id=settings.CERT_AWS_ID,
-                aws_key=settings.CERT_AWS_KEY,
-                dir_prefix=pdf_dir,
-                long_org=args.long_org,
-                long_course=args.long_course,
-                issued_date=args.issued_date,
-            )
-            title = args.assign_title or None
+            if not (cert and cert.is_reusable(course_id, designation)):
+                cert = CertificateGen(
+                    course,
+                    args.template_file,
+                    aws_id=settings.CERT_AWS_ID,
+                    aws_key=settings.CERT_AWS_KEY,
+                    dir_prefix=pdf_dir,
+                    long_org=args.long_org,
+                    long_course=args.long_course,
+                    issued_date=args.issued_date,
+                    designation=designation,
+                )
             grade = None
             if args.grade_text:
                 grade = args.grade_text
+
             (download_uuid, verify_uuid,
                 download_url) = cert.create_and_upload(name, upload=upload_files, copy_to_webroot=False,
-                                                       cleanup=False, designation=title, grade=grade)
+                                                       cleanup=False, grade=grade)
             certificate_data.append((name, course, args.long_org, args.long_course, download_url))
             gen_dir = os.path.join(cert.dir_prefix, S3_CERT_PATH, download_uuid)
             # Remove non-ascii chars from filename before saving locally (This is not the production filename)
             name = ''.join([i if ord(i) < 128 else ' ' for i in name])
-            copy_dest = '{copy_dir}/{course}-{name}.pdf'.format(
+            suffix = ''
+            if designation:
+                suffix = "_{designation}".format(
+                    designation=designation,
+                )
+            copy_dest = '{copy_dir}/{course}-{name}{suffix}.pdf'.format(
                 copy_dir=copy_dir,
+                course=course.replace("/", "-"),
                 name=name.replace(" ", "-").replace("/", "-"),
-                course=course.replace("/", "-"))
```

Comment:
Let's not alphabetize in this commit. 
Keep `course=` in the same place, and just add what you need (`suffix`) above it.